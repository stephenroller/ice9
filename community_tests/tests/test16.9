# an if statement			succeed

if true ->
 write "Hi There";
fi