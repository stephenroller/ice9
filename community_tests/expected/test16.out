Loading file: generated/test16.tm
1 
0 
Number of instructions executed = 33
